src/averageAges.js

```diff
@@ -15,11 +15,17 @@
  * @return {number}
  */
 function calculateMenAverageAge(people, century) {
-  // write code here
-  // learn how to use array methods like .filter .map .some .every .find .reduce
-  // avoid using loop and forEach
-  // replace `if ()` statement with &&, || or ?:
-  // without nesting
+  const men = people.filter(person => person.sex === 'm');
+
+  const requiredMen = century ? men.filter(person =>
+    Math.ceil(person.died / 100) === century) : men;
```

Comment:
all ternary operator parts should start on a new line, fix it everywhere
```suggestion
  const requiredMen = century 
    ? men.filter(person => Math.ceil(person.died / 100) === century) 
    : men;
```